lib/LocalVerifiableCredentialStore.js

```diff
@@ -57,29 +81,27 @@ export class LocalVerifiableCredentialStore {
    * Gets all verifiable credential instances that match the given parameters.
    *
    * @param {object} options - The options to use.
-   * @param {boolean} [options.displayable] - Flag used for if credential is
+   * @param {boolean} options.displayable - Flag used for if credential is
    *   displable or not.
-   * @param {string} [options.parentId] - ID of the parent credential.
-   * @param {string|Array} [options.type] - Type(s) of credential.
+   * @param {string} options.parentId - ID of the parent credential.
    *
    * @returns {Promise<Array>} List of matching VCs.
    */
-  async find({displayable, parentId, type}) {
+  async find({displayable, parentId}) {
     const selector = {
-      $and: [{'meta.profileId': this.profileId}]
+      'meta.profileId': this.profileId,
+      'meta.parentId': {$gte: null},
+      'meta.displayable': {$gte: null},
```

Comment:
Can't these just be left off? Or does that make the index not work?

Reply:
They need to be there or the indexes won't work.